cmd/operator-sdk/bundle/create.go

```diff
@@ -330,3 +333,57 @@ func copyScorecardConfig() error {
 	}
 	return nil
 }
+
+func addSDKMetricLabels() error {
+	var sdkMetricContent string
+	for key, value := range projutil.MakeBundleMetricsLabels() {
+		sdkMetricContent += fmt.Sprintf("LABEL %s=%s\n", key, value)
+	}
+	err := projutil.RewriteFileContents(bundle.DockerFile, "LABEL", sdkMetricContent)
+	if err != nil {
+		return fmt.Errorf("error rewriting dockerfile with metric labels, %v", err)
+	}
+	return nil
+}
+
+func rewriteBundleImageContents(outputDir, manifestDir string) error {
```

Comment:
A more consistent logical flow:
```go
func rewriteBundleImageContents(rootDir string) error {
    metricsLabels := projutil.MakeBundleMetricsLabels()

	// Write metrics labels to bundle.Dockerfile.
    if err := addLabelsToDockerfile(bundle.Dockerfile, metricsLabels); err != nil {
        return fmt.Errorf("error writing bundle image scorecard labels: %v", err)
    }

	// Write metrics annotations to annotations.yaml.
    annotationsFilePath := getAnnotationsFilePath(rootDir)
	err := addLabelsToAnnotations(annotationsFilePath, metricsLabels)
	if err != nil {
        return fmt.Errorf("error writing bundle scorecard annotations: %v", err)
	}

	// CopyScorecardConfig to bundle.Dockerfile. Do this last to ensure that the above operations were successful.
	if err := copyScorecardConfig(bundle.Dockerfile); err != nil {
		return fmt.Errorf("error adding image COPY directive for scorecard config: %v", err)
	}
return nil
```

I've renamed `writeMetricsToAnnotationsYaml -> addLabelsToAnnotations` and `addSDKMetricLabels -> addLabelsToDockerfile` for consistency.